shark/build.gradle

```diff
@@ -2,8 +2,8 @@ plugins {
   id("org.jetbrains.kotlin.jvm")
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_7
-targetCompatibility = JavaVersion.VERSION_1_7
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Doesn't this mean the emitted bytecode won't be java 1.7 compatible? Why change this? It's been a while, I forget.

Reply:
AGP 4.2 above default support java level is 8.